core/src/main/scala/cats/syntax/apply.scala

```diff
@@ -9,4 +9,19 @@ trait ApplySyntax extends TupleSemigroupalSyntax {
       val self = fa
       val typeClassInstance = F
     }
+
+  implicit final def catsSyntaxApplyOps[F[_], A](fa: F[A]): ApplyOps[F, A] =
+    new ApplyOps(fa)
+}
+
+final class ApplyOps[F[_], A](val fa: F[A]) extends AnyVal {
```

Comment:
Hmm, why is it that this new syntax is needed? Does simulacrum not generate syntax for deprecated methods?

Reply:
Generate as deprecated?

Reply:
Simulacrum does generate the deprecated methods, which makes scalac complain since it references the original deprecated method.